public/javascripts/chart/ratingHistory.js

```diff
@@ -26,7 +31,7 @@ lichess.ratingHistoryChart = function(data) {
           'Dash',
           // UltraBullet
           'ShortDot'
-        ];
+        ].filter((_, i) => !singlePerfName || i === singlePerfIndex);
```

Comment:
This is modern JS syntax, and this file is not compiled from typescript to standard JS. So I expect it will fail on many browsers.

Reply:
Hmm, you're right - https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Functions/Arrow_functions - seems like only IE doesn't support it, but as the README states that's still a supported browser. I'll get rid of that syntax tomorrow.